app/src/main/java/com/kylecombes/micrepeater/Setting.java

```diff
@@ -0,0 +1,44 @@
+package com.kylecombes.micrepeater;
+
+import android.content.Intent;
+import android.os.Bundle;

```

Comment:
Some of the imports are not needed (shown as gray color). Use `Alt + Enter` to "Optimize imports"